app/modules/visualization/partials/graph.html

```diff
@@ -54,18 +54,30 @@
 
     <center>
         <div ng-show="isSelectedSon('graph_pie')">
+        	<!--
             <div class="pcPieChart" dataset="dataset" chartid="visualization.id" show-Legend="showLegend" small="list" mode="mode" show-Labels="showLabels"></div>
+           	-->

```

Comment:
You shouldn't keep your old code in comments. If you need to, you can always check git history.